identity-core/Cargo.toml

```diff
@@ -32,11 +32,16 @@ features = ["ed25519", "random", "sha", "x25519"]
 js-sys = { version = "0.3.55", default-features = false }
 
 [dev-dependencies]
+criterion = { version = "0.3" }
 hex = { version = "0.4", default-features = false }
 proptest = { version = "1.0.0" }
 quickcheck = { version = "1.0" }
 quickcheck_macros = { version = "1.0" }
 
+[[bench]]
+name = "timestamp"
+harness = false
```

Comment:
Timestamp deserialization is not something likely to deteriorate over time nor a performance bottleneck, so I would argue against keeping a benchmark for it around.